tests/integration/test_charm.py

```diff
@@ -7,16 +7,18 @@
 
 log = logging.getLogger(__name__)
 
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
 
-@pytest.mark.abort_on_fail
-async def test_build_and_deploy(ops_test):
-    my_charm = await ops_test.build_charm(".")
-    resources = {"oci-image": "quay.io/dexidp/dex:v2.28.1"}
-    await ops_test.model.deploy(my_charm, resources=resources)
-    await ops_test.model.wait_for_idle()
 
-
-async def test_status(ops_test):
-    metadata = Path("./metadata.yaml")
-    charm_name = yaml.safe_load(metadata.read_text())["name"]
-    assert ops_test.model.applications[charm_name].units[0].workload_status == "active"
+# @pytest.mark.abort_on_fail
+# async def test_build_and_deploy(ops_test):
+#     my_charm = await ops_test.build_charm(".")
+#     image_path = METADATA["resources"]["oci-image"]["upstream-source"]
+#     resources = {"oci-image": image_path}
+#     await ops_test.model.deploy(my_charm, resources=resources)
+#     await ops_test.model.wait_for_idle()
+# 
+# 
+# async def test_status(ops_test):
+#     charm_name = METADATA["name"]
+#     assert ops_test.model.applications[charm_name].units[0].workload_status == "active"
```

Comment:
Why is this commented out?I think it's probably good to test both a fresh deploy as well as a local upgrade, but if not, you might as well remove the file entirely.  FWIW, `pytest-operator` should provide a separate model for each test module (though all tests w/in a given module share the same model).  This is what's causing the lint errors, as well, btw.

Reply:
I agree, this was accidentally left from when I was testing.